File: C_Programs/CheckEnteredNumberPrimeOrNot.cpp
//accept a number and print whether it is prime or not.

#include<stdio.h>
#include<conio.h>

int main()
{
	int num,cnt=2,rem;
	printf("Enter any Number : ");
	scanf("%d",&num);
	while(cnt<num)
	{
		rem=num%cnt;7
		if(rem==0)
		{
			break;
		}
		cnt++;
	}
	if (cnt==num)
	printf("\n Entered number is prime number");
	else
	printf("\nEntered number is not a prime number");
	return 0;
}

File: Basic_Java_Programs/CheckEnteredNumberPrimeOrNot.java
//check entered number is prime number or not
import java.util.*;

public class CheckEnteredNumberPrimeOrNot {
    public static void main(String[] args){
        int num,cnt=2,rem;
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter any Number : ");
        num=sc.nextInt();

        while(cnt<num) {
            rem=num%cnt;
            if(rem==0)
            {
                break;
            }
            cnt++;
        }
        if (cnt==num)
            System.out.println("Entered number is prime number");
        else
            System.out.println("Entered number is not a prime number");
    }
}
